steam/steam.c

```diff
@@ -202,6 +206,20 @@ static void steam_user_msg(SteamData *sata, SteamUserMsg *msg, gint64 time)
                        sid, msg->type, info->act);
 
     switch (msg->type) {
+    case STEAM_USER_MSG_TYPE_MY_EMOTE:
+    case STEAM_USER_MSG_TYPE_MY_SAYTEXT:
+        if (set_find(&sata->ic->bee->set, "self_messages") == NULL)
+            return;
+
+        if (msg->type == STEAM_USER_MSG_TYPE_MY_EMOTE)
+            str = g_strconcat("/me ", msg->text, NULL);
+        else
+            str = g_strdup(msg->text);
+
+		imcb_buddy_msg(sata->ic, sid, str, OPT_SELFMESSAGE, time);
```

Comment:
Didn't notice that. Fixed it. (I checked all other whitespace but apparently missed that line)